test/LEGO.AsyncAPI.Tests/Bindings/Pulsar/PulsarBindings_Should.cs

```diff
@@ -64,6 +66,46 @@ public void PulsarChannelBinding_WithFilledObject_SerializesAndDeserializes()
             binding.Should().BeEquivalentTo(channel);
         }
 
+        [Test]
+        public void PulsarChannelBindingNamespaceDefaultToNull()
+        {
+            // Arrange
+            var actual =
+                @"bindings:
+  pulsar:
+    persistence: persistent";
+
+            // Act
+            // Assert
+            var binding = new AsyncApiStringReader().ReadFragment<AsyncApiChannel>(actual, AsyncApiVersion.AsyncApi2_0, out _);
+
+            // Assert
+            Assert.AreEqual(null, ((PulsarChannelBinding)binding.Bindings[BindingType.Pulsar]).Namespace);
+        }
+
+        [Test]
+        public void PulsarChannelBindingPropertiesExceptNamespaceDefaultToNull()
+        {
+            // Arrange
+            var actual =
+                @"bindings:
+  pulsar:
+    namespace: staging"
+                ;
+
+            // Act
+            // Assert
+            var binding = new AsyncApiStringReader().ReadFragment<AsyncApiChannel>(actual, AsyncApiVersion.AsyncApi2_0, out _);
+
+            // Assert
+            Assert.AreEqual(null, ((PulsarChannelBinding)binding.Bindings[BindingType.Pulsar]).Persistence);
```

Comment:
Consider making this 
`(PulsarChannelBinding)binding.Bindings[BindingType.Pulsar])` a variable instead and using that - instead of having it multiple times.

Reply:
Good point, I'll fix that real quick